rollup.config.js

```diff
@@ -87,11 +87,9 @@ function createCommonJSConfig(input, output, options) {
       esModule: false,
       outro: options.addModuleExport
         ? [
-            `module.exports = ${options.addModuleExport.default};`,
```

Comment:
The changes in the rollup config result in a change in the `build/index.js` file. I'm confident the first line can be removed but less confident the last line can be. @dai-shi, please take a look and let me know what you find. 

```diff
- module.exports = react;
module.exports.create = create;
module.exports.useStore = useStore;
module.exports.createStore = vanilla.createStore;
- exports.default = module.exports;
```

Reply:
We don't need `addModuleExport` entirely. That was a workaround for default exports and a big motivation to drop it in v5.
Oh, this is hard without knowing the context.

Reply:
Ah, that makes sense. I see why it can be removed now. I learned something new about CJS. Thanks!